example/reactweb/src/App.tsx

```diff
@@ -42,52 +36,41 @@ function ShowError(props: { error: unknown }) {
 function Root() {
   const initialClientID = readClientID();
   const initialEndpoint = readEndpoint();
-  const initialTransientSession = readTransientSession();
   const [clientID, setClientID] = useState(initialClientID);
   const [endpoint, setEndpoint] = useState(initialEndpoint);
-  const [transientSession, setTransientSession] = useState(
-    initialTransientSession
-  );
+
   const [error, setError] = useState<unknown>(null);
   const [userInfo, setUserInfo] = useState<unknown>(null);
 
-  const configure = useCallback(
-    (clientID: string, endpoint: string, transientSession: boolean) => {
-      window.sessionStorage.setItem("authgear.demo.clientID", clientID);
-      window.sessionStorage.setItem("authgear.demo.endpoint", endpoint);
-      window.sessionStorage.setItem(
-        "authgear.demo.transientSession",
-        transientSession ? "true" : "false"
+  const configure = useCallback((clientID: string, endpoint: string) => {
+    window.sessionStorage.setItem("authgear.demo.clientID", clientID);
+    window.sessionStorage.setItem("authgear.demo.endpoint", endpoint);
```

Comment:
Can we update to use `localStorage` instead? That would be more useful :)

Reply:
Sure, updated!:)